notebooks/validation_notebooks/slsim_dc2_diffsky_galaxy_distribution_comparision.ipynb

```diff
@@ -0,0 +1,758 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #85.    plt.show()
</pre><p>can you cut these plots at mag=30 to better see what's in the more relevant part of the magnitude range. Can you also make a plot where you show the evolution as a function of redshift in one plot? By eye, I can't even see an evolution in the luminosity function as a function of redshift for the SkyPy distributions, which makes me worried</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>